README.md

```diff
@@ -105,6 +105,18 @@ $ rdf-test-suite myengine.js http://w3c.github.io/rdf-tests/sparql/sparql11/mani
   -t test_pn_bad_0
 ```
 
+### Test skipping
+
+With the `--skip` option you can filter tests that should be skipped
+based on a regex that will be matched with the test URI.
+
+For example:
+
+```bash
+$ rdf-test-suite myengine.js http://w3c.github.io/rdf-tests/sparql/sparql11/manifest-all.ttl \
+  --skip test_pn_bad_0
```

Comment:
Could we add an example of a case where we need to skip 2 tests?
Just so that people can just copy-paste that format in such cases.

Reply:
Would you like it to be with the `(a|b)` notation or the `a.*`. Probably `a|b`?

Reply:
`(a|b)` might be better, as it's more easily extensible.